src/Tasks/Microsoft.NET.Build.Tasks/GenerateDepsFile.cs

```diff
@@ -50,6 +52,9 @@ public class GenerateDepsFile : TaskBase
         [Required]
         public ITaskItem[] ReferenceSatellitePaths { get; set; }
 
+        [Required]
+        public ITaskItem[] FilesToSkip { get; set; }
```

Comment:
We should probably have a test that verifies the .deps.json file is trimmed correctly.  I don't see one.  It could probably go into Microsfot.NET.Build.Tests